packages/wallets/solflare/src/solflare.ts

```diff
@@ -1,47 +1,26 @@
-import type {
-    ConfirmOptions,
-    Connection,
-    PublicKey,
-    SendOptions,
-    Signer,
-    Transaction,
-    TransactionSignature,
-} from '@solana/web3.js';
+import type { Connection, PublicKey, Transaction } from '@solana/web3.js';
 
-export interface BackpackWindow extends Window {
-    backpack: Backpack;
+export interface SolflareWindow extends Window {
+    solflare: Solflare;
 }
 
-export interface BackpackEvents {
+export interface SolflareEvents {
     connect(): void;
     disconnect(): void;
     connectionDidChange(): void;
 }
 
-export interface Backpack {
+export interface Solflare {
     isConnected: boolean;
     publicKey: PublicKey | undefined;
-    connection: Connection;
+    connection: Connection | undefined;
     connect(): Promise<void>;
     disconnect(): Promise<void>;
-    sendAndConfirm(
-        tx: Transaction,
-        signers?: Signer[],
-        options?: ConfirmOptions,
-        connection?: Connection,
-        publicKey?: PublicKey
-    ): Promise<TransactionSignature>;
-    send(
-        tx: Transaction,
-        signers?: Signer[],
-        options?: SendOptions,
-        connection?: Connection,
-        publicKey?: PublicKey
-    ): Promise<TransactionSignature>;
```

Comment:
i noticed that solflare doesn't have `sendAndConfirm` and `send` methods

what could be a substitution of those?
maybe you guys can provide a hint @jordansexton @olistic 🙏 

<img width="465" alt="Screen Shot 2022-09-17 at 14 53 32" src="https://user-images.githubusercontent.com/26966797/190847190-7476b1d4-288c-4fab-bfa6-cf1ea3ce2720.png">

Reply:
According to https://github.com/solflare-wallet/solflare-sdk#api Solflare doesn't have a method to sign and send transactions, so you would need to implement this by signing using the wallet and then sending within an adapter layer.